nvflare/private/fed/server/client_manager.py

```diff
@@ -141,14 +141,17 @@ def authenticated_client(self, request, context) -> Optional[Client]:
                 )
                 return None
 
-            for token, client in self.clients.items():
-                if client.name == client_name:
-                    with self.lock:
-                        self.clients.pop(token)
+            with self.lock:
+                clients_to_be_moved = []
```

Comment:
suggest renaming this to `clients_to_be_removed = []`

Reply:
changed.